client/Images/imagesTypes.d.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+declare module '*.png' {
```

Comment:
I think with other changes having gone in, this has been superceeded by `Images.d.ts`?

Reply:
works for me I'll get rid of it.